src/gic_calculator/gic_difference.py

```diff
@@ -0,0 +1,46 @@
+from gic_calculator.interest_calculator import interest_calc
+
+def calculate_gic_difference(term_length_n1, term_length_n2, principal, interest_rate1=None, interest_rate2=None):
```

Comment:
Is there value in standardizing the input format with the gic_plotting function? I.e.:
gic_plotting(principal, term_lengths, gic_rates=None)
term_lengths : list
 gic_rates : list, optional
 ex/ gic_plotting(interests=[17.26, 96.16], term_lengths=[90, 180])